github/compliance/model.py

```diff
@@ -119,15 +119,17 @@ def calculate_latest_processing_date(
         if delivery_svc_client and repository and max_days > 0:
             try:
                 import github.compliance.report as gcr
```

Comment:
Even though not introduced with this PR, why late import?
If it is just to avoid circular imports, I would rather consider to factor out this function to another module.

Reply:
A good pattern to avoid circular imports is to have `model` classes not dependent on other modules. Same goes for `util` modules by the way.
These should be importable in any location.

Reply:
yep, was to avoid circular imports. Will add a commit which moves affected functions to a different module